packages/komencikit/README.md

```diff
@@ -0,0 +1,29 @@
+# KomenciKit
+
+The KomenciKit library is a wrapper for the Komenci Service API, which is used for fee-less onboarding by allowing 
+the consumer to execute fee-less attestations with the help of MetaTransactions.
+
+The main actions that KomenciKit exposes are:
+- startSession: Initiate a Komenci session with a solved captcha
+- deployWallet: deploy a MTW for Valora's EOA
+- requestAttestations: request attestations fee-lessly 
+- submitMetaTransaction: submit arbitrary meta transactions via Komenci
```

Comment:
Do we want to allow this? For security, each contract and method should be added manually and building an API to expose it is a good forcing function to perform session checks and input validation.

Reply:
I get what you're saying. I guess to I saw this as a way to reduce duplication and simplify the logic where the meta-tx is simple and we don't need to add anything on top it could be passed in directly. This would be coupled with server-side validation to only filter allowed calldata and ensure that the wallet called is what is expected.

But I can also see the point of having a clearer KomenciKit API, even if the exposed functions rely on a generalised end-point on Komenci's side.

Reply:
Or I'm not sure if you're referring to also exposing these individually on the Komenci API side as well or not. I guess it would potentially make it easier to validate if we do that.